website/client/js/controllers/rootCtrl.js

```diff
@@ -6,6 +6,9 @@
 habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$http', '$state', '$stateParams', 'Notification', 'Groups', 'Shared', 'Content', '$modal', '$timeout', 'ApiUrl', 'Payments','$sce','$window','Analytics','TAVERN_ID',
   function($scope, $rootScope, $location, User, $http, $state, $stateParams, Notification, Groups, Shared, Content, $modal, $timeout, ApiUrl, Payments, $sce, $window, Analytics, TAVERN_ID) {
     var user = User.user;
+    var IGNORE_SCROLL_PAGES = {
+        'options.social.challenges.detail': true
```

Comment:
When you click edit it jumps to top, back to challenges doesn't. But when you get back to the challenges page it won't be scrolled down to the challenge you were editing if it's a long way down. But that's because the back to challenges button is on the top of the page, not sure if that's something weird.